.helm/ecamp3/templates/mail_ingress.yaml

```diff
@@ -7,23 +7,23 @@ metadata:
   labels:
     {{- include "mail.selectorLabels" . | nindent 4 }}
     {{- include "app.commonLabels" . | nindent 4 }}
-  {{- with .Values.ingress.annotations }}
   annotations:
+    {{- with .Values.ingress.annotations }}
     {{- toYaml . | nindent 4 }}
-  {{- end }}
+    {{- end }}
```

Comment:
Why this change? Is the annotations key required now for some reason?

Reply:
Because i started with rewriting the url (like for api or print), but then i did not need it.
If you need to append annotations which are not in the values, then you need to change it.
Should i revert this change, or also apply it to frontent_ingress.yml?